docs/content/preview/migrate/troubleshoot.md

```diff
@@ -416,6 +462,73 @@ PRIMARY KEY (employee_id, hire_date)) PARTITION BY RANGE (hire_date) ;
 
 ---
 
+### Issue #612: [PostgreSQL] Adding primary key to a partitioned table results in an error
+
+**GitHub link**: [Issue #612](https://github.com/yugabyte/yb-voyager/issues/612)
+
+**Description**: If you have a partitioned table in which primary key is added later using `ALTER TABLE`, then it fails with the following error:
```

Comment:
then it fails -> then the migration fials?

Reply:
table creation fails . Ill update it.